### PR TITLE
feat(deploy-tooling): 1821 change TR description

### PR DESCRIPTION
**Pull request by korotkovao**
https://github.com/SAP/open-ux-tools/issues/1821

Change the default TR description to include the name of the ABAP repo used in deploy config so that users can more easily see what the TR relates to. The suggested new default text should be:

`Created by SAP Open UX Tools for ABAP repository <repository name>`

### CODE REVIEW

File: packages/deploy-tooling/test/unit/base/deploy.test.ts [hunk 1]
```diff
@@ -233,6 +233,9 @@ describe('base/deploy', () => {
             });
             expect(config.createTransport).toBe(false);
             expect(mockedAdtService.createTransportRequest).toBeCalledTimes(1);
+            expect(mockedAdtService.createTransportRequest).toBeCalledWith(
+                expect.objectContaining({ description: 'Created by SAP Fiori tools for ABAP repository ~name' })
```
Comment (longieirl) [inline packages/deploy-tooling/test/unit/base/deploy.test.ts:237]: We need to re-consider this sentence, as its misleading.

Comment (korotkovao) [reply packages/deploy-tooling/test/unit/base/deploy.test.ts]: Thank you, it has been updated.

**Review by longieirl — APPROVED** [review]
Changes look good.
Tests to cover change.

**Review by docirl — APPROVED** [review]
Changes look good and covered by test

**Pull request merged**